docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -0,0 +1,325 @@
+   when it is activated, because the generated service file will never try to download images.
+   It is recommended to use a fully qualified image name rather than a short name both for
+   performance and robustness reasons.
+
+#### `ContainerName=`
+
+   The (optional) name of the podman container. If this is not specified, the default value
+   of `systemd-%N` will be used, which is the same as the service name but with a `systemd-`
+   prefix to avoid conflicts with user-managed containers.
+
+#### `Environment=`
+
+  Set an environment variable in the container. This uses the same format as
+  [services in systemd](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Environment=)
+  and can be listed multiple times.
+
+#### `Exec=`
+
+  If this is set then it defines what commandline to run in the container. If it  is not set the
+  default entry point of the container image is used.  The format is the same as for
+  [systemd command lines](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Command%20lines).
+
+#### `User=`
+
+  The (numeric) uid to run as inside the container. This does not need to match the uid on the host,
+  which can be set with `HostUser`, but if that is not specified the uid is used on the host.
+
+#### `HostUser=`
+
+  The host uid (numeric or a username) to run the container as. If this differs from the uid in `User`
+  then user namespaces are used to map the ids. If unspecified this defaults to what was specified in `User`.
+
+#### `Group=`
+
+  The (numeric) gid to run as inside the container. This does not need to match the gid on the host,
+  which can be set with `HostGroup`, but if that is not specified the same host gid is used.
+
+#### `HostGroup=`
+
+  The host gid (numeric or group name) to run the container as. If this differs from the gid in `Group`
+  then user namespaces are used to map the ids. If unspecified this defaults to what was specified in `Group`.
+
+#### `NoNewPrivileges=` (defaults to `yes`)
+
+  If enabled (which is the default) this disables the container processes from gaining additional privileges via things like
+  setuid and file capabilities.
+
+#### `DropCapability=` (defaults to `all`)
+
+   Drop these capabilities from the default container capability set. The default is `all`, so you
+   have to add any capabilities you want with `AddCapability`. Set this to empty to drop no capabilities.
+   This can be listed multiple times.
+
+#### `AddCapability=`
+
+   By default the container runs with no capabilities (due to DropCapabilities='all'), if any specific
+   caps are needed you can add them with this key. For example using `AddCapability=CAP_DAC_OVERRIDE`.
+   This can be listed multiple times.
+
+#### `ReadOnly=` (defaults to `no`)
+
+   If enabled, makes image read-only, with /var/tmp, /tmp and /run a tmpfs (unless disabled by `VolatileTmp=no`).
```

Comment:
Could you add something like
Note: Podman will automatically copy content from the image onto the tmpfs.